src/js/video.js

```diff
@@ -810,5 +810,10 @@ videojs.dom = Dom;
  */
 videojs.url = Url;
 
+/**
+ * Replaces format-time with a custom implementation
```

Comment:
can the JSdoc be elaborated here? This is the main doc that Video.js users will see for this method, probably.
Also, we should mention that if an item, like the duration, has already formatted itself, setting a new format time won't update that item.